Source/DafnyLanguageServer.Test/Diagnostics/VerificationDiagnostics.cs

```diff
@@ -13,6 +13,36 @@ namespace Microsoft.Dafny.LanguageServer.IntegrationTest.Synchronization;
 
 public class VerificationDiagnostics : ClientBasedLanguageServerTest {
 
+  [Fact]
+  public async Task SubExpression() {
```

Comment:
```suggestion
  public async Task SubProposition() {
```

Really a nit pick, feel free to ignore if you can merge earlier.